client/my-sites/domains/style.scss

```diff
@@ -10,3 +11,45 @@
 		align-items: center;
 	}
 }
+
+body.is-section-domains.is-experiment-user {
+	background: #fdfdfd;
+
+	.domains__header {
+		display: block;
+
+		h1 {
+			font-family: $brand-serif;
+			font-size: $font-headline-small;
+		}
+
+		p {
+			font-size: $font-body;
+			color: var(--color-text-subtle);
+			text-align: center;
+			padding: 0 10px;
+		}
+
+		a {
+			color: var(--studio-gray-90);
+			font-weight: 500;
+			padding-left: 5px;
```

Comment:
Do we really want to add a left space to every link? This seems like a code smell.

Reply:
It would be applied only at the experiment header, but assuming this experiment will be successful we'll make this code permanent, so good point, [I improved it here](https://github.com/Automattic/wp-calypso/pull/73017/commits/7f91d2e9e3df70265ae00ba0722efecaaf4e67b9).